I update CONTRIBUTING docs.

SweetmanTech opened the pull request.

amaredeus commented on docs/CONTRIBUTING.md:
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
-1. Create a new feature branch from the dev branch
-2. Copy the .env file and name it whatever you want; add it to your local
-   .gitignore
+1. Create a new feature branch from the dev branch (`docs/<name>`, `feature/<name>`, `release/<name>`, `patch/<name>`).
```
We opted to go with `hotfix/<name>` instead of `patch/<name>`

SweetmanTech replied:
updated ✅

amaredeus commented on docs/CONTRIBUTING.md:
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
    file, making sure to update the file name where it is required in `app.js`
-4. `npm install`
-5. Run your bot and test your connections
+4. `yarn`
```
`yarn install` or even simpler doing `yarn start` will also run the prestart script to get everything installed and running

SweetmanTech replied:
updated ✅

amaredeus reviewed (changes requested):
Awesome work on the jobs, just small minor suggestions

amaredeus commented:
@SweetmanTech Looks like build is failing because branch SweetmanTech:docs/onboarding is not in sync with Bankless:develop branch. Can you try running 

`git pull origin develop` where origin is git url from Bankless?

SweetmanTech commented:
Addressed comments ✅ 

sorry for the delay @nonsensecodes & @amaredeus I just finished moving back to the US from Thailand and as soon as I got back I got COVID but I'm recovering now and getting back up to speed with everything!

Let me know if there's any other feedback I can address here. 🙏

amaredeus commented on docs/CONTRIBUTING.md:
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
 -   View Channels
 -   All of the permissions under "Text Permissions"
--   No permissions under "Voice Permissions"
+-   Only permissions under "Voice Permissions" are Connect & Use Voice Activity.
```
Voice permissions are not  needed. We can revert this change

amaredeus commented on docs/CONTRIBUTING.md:
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
 -   View Channels
 -   All of the permissions under "Text Permissions"
--   No permissions under "Voice Permissions"
+-   Only permissions under "Voice Permissions" are Connect & Use Voice Activity.
```
Voice permissions are not needed. We can revert this change

SweetmanTech replied:
rolled back changes here ✅

amaredeus commented on docs/CONTRIBUTING.md:
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
 
 10. Select and copy the generated URL under "Scopes." Make sure the value
-    of permissions in the URL reads 4027055222
+    of permissions in the URL reads 536637602902.
```
since this value can change based on needs, can we have a link here to the PERMISSIONS file in docs?

SweetmanTech replied:
great idea 💡

SweetmanTech replied:
updated ✅

nonsensecodes reviewed (approved):
As it appears the requested changes were implemented, I'm approving this PR. Thank you @SweetmanTech for following through and making the edits, and thank you @amaredeus for reviewing and suggesting said changes. Great job!
